apps/range-voting/app/script.js

```diff
@@ -117,98 +118,93 @@ async function loadVoteDescription(vote) {
 }
 
 async function loadVoteData(voteId) {
-  console.info('[RangeVoting > script]: loadVoteData')
+  console.info('[RangeVoting > script]: loadVoteData', voteId)
   let vote
   return new Promise((resolve, reject) => {
     app
       .call('getVote', voteId)
-      .first()
-      .subscribe(voteData => {
+      .pipe(first())
+      .subscribe(async voteData => {
         let funcSig = voteData.executionScript.slice(58, 66)
         if (funcSig == 'f2122136') {
           console.log('Loading Projects Data')
           resolve(loadVoteDataProjects(voteData, voteId))
         } else {
           console.log('Loading Allocations Data')
+          console.log('vote data: ', await loadVoteDataAllocation(voteData, voteId))
```

Comment:
I don't think that pushing this anonymous function to become async is worth being able to log it here. If anything if you want this info I'd log it out in like handleEvents or updateState after the updateVotes call?

Reply:
agreed, would love to take out all of these logs. I kept it in there and added the `await` keyword because there was a log in there that called `loadVoteDataAllocation` but it didn't await, so the log was useless. I'll take it out all together